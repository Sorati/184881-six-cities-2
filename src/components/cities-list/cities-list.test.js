import renderer from 'react-test-renderer';
import CitiesList from './cities-list';

it(`Cities list correctly renders with mock cities after relaunch`, () => {
  const tree = renderer
    .create(<CitiesList
      cities={[`Amsterdam`, `Moscow`]}
      currentCity={`Moscow`}
      onCityFilterClick={jest.fn()}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
