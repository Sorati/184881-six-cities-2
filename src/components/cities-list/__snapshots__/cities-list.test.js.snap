// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cities list correctly renders with mock cities after relaunch 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item false"
      href="#"
      onClick={[Function]}
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
      onClick={[Function]}
    >
      <span>
        Moscow
      </span>
    </a>
  </li>
</ul>
`;
